toolkit/tools/imager/imager.go

```diff
@@ -403,18 +403,28 @@ func fixupExtraFilesIntoChroot(installChroot *safechroot.Chroot, config *configu
 	return
 }
 
-func cleanupExtraFilesInChroot(installChroot *safechroot.Chroot, config configuration.SystemConfig) (err error) {
+func cleanupExtraFiles() (err error) {
 	dirsToRemove := []string{additionalFilesTempDirectory, postInstallScriptTempDirectory, sshPubKeysTempDirectory}
+
 	for _, dir := range dirsToRemove {
+		logger.Log.Infof("Cleaning up directory %s", dir)
 		err = os.RemoveAll(dir)
 		if err != nil {
-			logger.Log.Errorf("Failed to cleanup directory (%s). Error: %s", dir, err)
+			logger.Log.Warnf("Failed to cleanup directory (%s). Error: %s", dir, err)
 			return
 		}
 	}
 	return
 }
 
+func cleanupExtraFilesInChroot(chroot *safechroot.Chroot) (err error) {
+	logger.Log.Infof("Proceeding to cleanup extra files in chroot %s.", chroot.RootDir())
```

Comment:
>logger.Log.Infof("Proceeding to cleanup extra files in chroot %s.", chroot.RootDir()) [](start = 1, length = 85)

How often is this called during one build? If it's more than once, I'd suggest moving this log to the debug level (same for `cleanupExtraFiles`). If it's always at most once, then please ignore this comment.

Reply:
Anyone could call this, but currently it's called only once.